s32/mcux/devices/S32Z270/S32Z270_features.h

```diff
@@ -110,6 +110,135 @@
 #define FSL_FEATURE_FLEXCAN_HAS_NO_SLFWAK_SUPPORT (1)
 /* @brief Does not support Wake Up Source (bitfield MCR[WAKSRC]. */
 #define FSL_FEATURE_FLEXCAN_HAS_NO_WAKSRC_SUPPORT (1)
+/* @brief EDMA availability on the SoC. */
+#define FSL_FEATURE_SOC_EDMA_COUNT (5)
+
+/* EDMA module features */
+
+/* @brief Number of DMA channels (related to number of registers TCD, DCHPRI, bit fields ERQ[ERQn], EEI[EEIn], INT[INTn], ERR[ERRn], HRS[HRSn] and bit field widths ES[ERRCHN], CEEI[CEEI], SEEI[SEEI], CERQ[CERQ], SERQ[SERQ], CDNE[CDNE], SSRT[SSRT], CERR[CERR], CINT[CINT], TCDn_CITER_ELINKYES[LINKCH], TCDn_CSR[MAJORLINKCH], TCDn_BITER_ELINKYES[LINKCH]). (Valid only for eDMA modules.) */
+#define FSL_FEATURE_EDMA_MODULE_CHANNEL (32)
+/* @brief If 8 bytes transfer supported. */
+#define FSL_FEATURE_EDMA_SUPPORT_8_BYTES_TRANSFER (1)
+/* @brief Number of channel for each EDMA instance, (only defined for soc with different channel numbers for difference instance) */
+#define FSL_FEATURE_EDMA_INSTANCE_CHANNELn(x) (32)
```

Comment:
how this relates to https://github.com/zephyrproject-rtos/zephyr/pull/78368/files#diff-9ae4a85169b9753909d6b33ec1185b4e689e9d1a76182de5458cd9b9a4601a86R171-R172 ?

Reply:
Yes, I have updated this definition, instance 0 (32) and other (16).